LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Ryan Rhee
+Copyright (c) 2021 Max Winterstein
```

Comment:
by the way, if you're making a derivative work you need to include the original license to abide by its terms

Reply:
thanks for pointing at this, adapted some from [stackexchange](https://softwareengineering.stackexchange.com/questions/277688/if-i-fork-a-project-on-github-that-is-licensed-under-mit-how-do-i-handle-the-at) and hope this will do the job.

btw, thanks for your work at all this stuff :)